internal/backend/sessions/sessionworkflows/module.go

```diff
@@ -0,0 +1,61 @@
+package sessionworkflows
+
+import (
+	"context"
+	"fmt"
+
+	"go.autokitteh.dev/autokitteh/internal/backend/fixtures"
+	"go.autokitteh.dev/autokitteh/sdk/sdkexecutor"
+	"go.autokitteh.dev/autokitteh/sdk/sdkmodule"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+func (w *sessionWorkflow) newModule() sdkexecutor.Executor {
+	flags := sdkmodule.WithFlags(
+		sdktypes.PureFunctionFlag,           // no need to run in an activity, we must have it in the workflow.
+		sdktypes.PrivilidgedFunctionFlag,    // provide workflow context.
+		sdktypes.DisablePollingFunctionFlag, // no polling.
+	)
+
+	return fixtures.NewBuiltinExecutor(
+		fixtures.ModuleExecutorID,
+		sdkmodule.ExportValue("timeout_error", sdkmodule.WithValue(fixtures.TimeoutError)),
+		sdkmodule.ExportFunction("syscall", w.syscall, flags),
+		sdkmodule.ExportFunction("poll", w.setPoller, flags),
+		sdkmodule.ExportFunction("fake", w.fake, flags),
+		sdkmodule.ExportFunction("sleep", w.sleep, flags),
+		sdkmodule.ExportFunction("start", w.start, flags),
+		sdkmodule.ExportFunction("subscribe", w.subscribe, flags),
+		sdkmodule.ExportFunction("unsubscribe", w.unsubscribe, flags),
+		sdkmodule.ExportFunction("next_event", w.nextEvent, flags),
+		sdkmodule.ExportFunction("callopts", callopts, flags),
+		sdkmodule.ExportFunction("is_deployment_active", w.isDeploymentActive),
+	)
+}
+
+func callopts(_ context.Context, args []sdktypes.Value, kwargs map[string]sdktypes.Value) (sdktypes.Value, error) {
+	if len(args) > 0 {
+		return sdktypes.InvalidValue, fmt.Errorf("expecting only key value arguments")
+	}
+
+	return sdktypes.NewStructValue(sdktypes.NewSymbolValue(fixtures.CallOptsCtorSymbol), kwargs)
+}
+
+func (w *sessionWorkflow) isDeploymentActive(ctx context.Context, args []sdktypes.Value, kwargs map[string]sdktypes.Value) (sdktypes.Value, error) {
```

Comment:
I'd still like to call "native" Go function

```
if cbs.IsDeploymentActive() {
    // ...
}
```

And not do the boxing/unboxing dance.

Reply:
this will happen in a following pr. this is just a prep for that.